### PR TITLE
feat: add support for deno as package manager

### PR DESCRIPTION
**Pull request by bartlomieju**
Resolves https://github.com/unjs/undocs/issues/146.

### CODE REVIEW

File: app/utils/package-manager.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ export const packageManagers = [
   { name: 'yarn', command: 'yarn', install: 'add', run: '', x: 'yarn dlx' },
   { name: 'pnpm', command: 'pnpm', install: 'i', run: '', x: 'pnpm dlx' },
   { name: 'bun', command: 'bun', install: 'i', run: 'run ', x: 'bunx' },
+  { name: 'deno', command: 'deno', install: 'i', run: 'run ', x: 'deno run -A npm:' },
```
Comment (bartlomieju) [inline app/utils/package-manager.ts:6]: This is really not correct, as the `x` will produce a command like this:
```
deno run -A npm: giget gh:unjs/undocs/template docs --install
```
(notice the space after `npm:`)

Is there an easy way we could handle this problem?

Comment (pi0) [reply app/utils/package-manager.ts:6]: should be fixed with [`99e329b` (#147)](https://github.com/unjs/undocs/pull/147/commits/99e329b11f15a4b5799df415d194997ad2ae8a4c)

**Review by pi0 — COMMENTED** [review]
Thanks!

**Comment by bartlomieju** [comment]
Thanks!

**Pull request merged**